Makefile

```diff
@@ -29,26 +32,28 @@ test: development
 
 requirements: requirements.txt requirements-dev.txt
 
-requirements.txt: pyproject.toml
-	$(VENV)/bin/pip-compile -v --resolver=backtracking --output-file=$@ $<
```

Comment:
why is the --resolver=backtracking not needed anymore?
Is [this](https://github.com/jazzband/pip-tools/issues/1868) not needed anymore?